app/ui/src/app/connections/list/list.component.html

```diff
@@ -108,24 +108,17 @@ <h1>Register Applications</h1>
         </div>
       </ng-container>
       <ng-container *ngIf="showNewConnection && connections.length > 0">
-        <div class="connection new-connection col-xs-12 col-sm-3 col-md-2 card"
-             (click)="onSelect(undefined, $event)">
-          <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
-            <div class="card-pf-body">
-
-              <!-- Card Icon -->
-              <div class="card-pf-top-element">
-                <span class="card-pf-icon-large pficon pficon-add-circle-o">
-                </span>
-              </div>
-
-              <!-- Card Description / Item Overview -->
-              <div class="card-pf-info text-center">
-                <button class="btn btn-default" (click)="onSelect(undefined, $event)">Create Connection</button>
-              </div>
-            </div>
-          </div>
-        </div>
+        <syndesis-empty-state-card
+          [body]="cardBody"
+          [colClass]="'col-xs-12 col-sm-3 col-md-2'"
+          [topElement]="cardTopElement">
+          <ng-template #cardBody>
+            <button class="btn btn-default" (click)="onSelect(undefined, $event)">Create Connection</button>
```

Comment:
If all we need is to customize the button text, make the text an input instead of this whole template.  Then you can define this button in your component and have the `(click)` trigger the `onSelect` function you've defined in this component.

Reply:
this is how the overall tempalte would become with all the above changes:

```html
<div class="empty-state-card card card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
  <div class="card-pf-body">
    <div class="card-pf-top-element">
      <ng-content></ng-content>
    </div>
    <div class="card-pf-info text-center">
      <button class="btn btn-default" (click)="onClick.emit($event)">{{ buttonLabel }}</button>
    </div>
  </div>
</div>
```

Please note that `<ng-content></ng-content>` now features no selector, since we will expect to have just one transcluded slot in the component definition:

```html
<syndesis-empty-state-card class="col-xs-12 col-sm-3 col-md-2">
  <!-- The specific HTML for EACH empty state card -->
</syndesis-empty-state-card>
```